scripts/doindex/tasks/records.mjs

```diff
@@ -0,0 +1,38 @@
+import { loadFile } from '../lib/html.mjs';
+import * as PARSERS from '../parsers/index.mjs';
+
+function log(type, ...args) {
+    process.send({ event: 'log', type, data: args });
```

Comment:
When logging happens in multiple thread happens this:
<img width="783" alt="Screenshot 2024-07-12 at 15 00 18" src="https://github.com/user-attachments/assets/07276bac-f4c4-4019-b4d1-4a5f7a02ff6b">

That's why logging only in main thread.